### ISSUE TITLE
Editor Logo not consistent across color schemes

### ISSUE THREAD

**Issue by Seacle** [focus-editor/focus#377]
In some of the color schemes for the editor the logo is missing the inner, the outer or both circles. I assume this is because the circle color is determined by color scheme colors generally, but it is kind of a bad look when first scrolling through the default color schemes so maybe change that. Not a big deal, but would like to have pointed it out.

**Comment by focus-editor** [comment]
Yes the logo colors are taken from the background colors which are user-provided. We haven't considered this to be a bug, but during a future polish run we might do something to make the logo outlines still visible, or do something else. Thanks for pointing it out.
